resources/views/admin/question/show.blade.php

```diff
@@ -27,16 +27,68 @@
         </div>
       </div>
 
-      <div id="chart" style="height: 300px;"></div>
-
+      <canvas id="bar-chart" style="height: 300px;"></canvas>
+      <canvas id="pie-chart" style="height: 300px;"></canvas>
     </div>
   </div>
 </div>
+
 <script>
+let bar_options = {
+  title: {
+    display: true,
+    text: 'Number of Users'
+  },
+  legend: {
+    display: false,
+  },
+}
+
+let pie_options = {
+  title: {
+    display: true,
+    text: 'Percentage of users',
+  },
+  tooltips: {
+    callbacks: {
+      label: function(tooltipItem, data) {
+        let dataArray = data.datasets[tooltipItem.datasetIndex].data || '';
+        let index = tooltipItem.index;
+
+        let percentageArray = [];
+        let sum = 0;
+        dataArray.map(data => {
+          sum += data;
+        });
+
+        dataArray.map(data => {
+          percentageArray.push((data * 100 / sum).toFixed(2));
+        });
```

Comment:
I wonder if we could calculate the percentages in PHP and add it as a custom attribute on dataset. In general, I would push the logic to PHP where it is easier to get the code tested, or we can try to figure out how to test the Javascript.